src/platform/forms-system/test/js/components/FormNav.unit.spec.jsx

```diff
@@ -1,12 +1,12 @@
-import React from 'react';
 import { expect } from 'chai';
+import React from 'react';
 import SkinDeep from 'skin-deep';
-
 import FormNav from '../../../src/js/components/FormNav';
 
 describe('Schemaform FormNav', () => {
-  it('should render current chapter data', () => {
-    const formConfig = {
+  let formConfigDefaultData;
+  before(() => {
+    formConfigDefaultData = {
```

Comment:
This works fine, but another pattern I've used in the past is to define the re-used variable (`formConfigDefaultData`) each time in the scope of the `it()` callback, but get the value from a function which returns a constant instead of redefining it in `before()`.

```js
const getDefaultData = () => ({ ... });

it('should render current chapter data', () => {
  const formConfigDefaultData = getDefaultData();
  ...
});
```